test/powershell/Modules/Microsoft.PowerShell.Utility/Get-FileHash.Tests.ps1

```diff
@@ -3,8 +3,9 @@
 Describe "Get-FileHash" -Tags "CI" {
 
     BeforeAll {
-        $testDocument = Join-Path -Path $PSScriptRoot -ChildPath assets testablescript.ps1
-        Write-Host $testDocument
+        $testDocument = Join-Path -Path $TestDrive -ChildPath "hashable.txt"
+        # Encode bytes of "<BOM>Get-Module`n" directly to avoid platform issues
+        [System.IO.File]::WriteAllBytes($testDocument, @(0xef, 0xbb, 0xbf, 0x47, 0x65, 0x74, 0x2d, 0x4d, 0x6f, 0x64, 0x75, 0x6c, 0x65, 0x0a))
```

Comment:
Based on your explanation, we are moving towards BOM-less UTF-8, so perhaps we should remove the BOM, get the bytes at runtime making the code more readable, and updating the expected hash for validation.